atlasdb-config/src/main/java/com/palantir/atlasdb/config/AtlasDbConfig.java

```diff
@@ -305,6 +305,11 @@ public boolean enableNonstandardAndPossiblyErrorProneTopologyAllowDifferentKvsAn
         return false;
     }
 
+    @Value.Default
```

Comment:
can we add a bit of docs on what exactly this does? I think we may want to name it `collectThreadDumpOn*TimestampService*Init` or something like that, as well